remote/hackillinois-osf-2021/OWNERS

```diff
@@ -0,0 +1,6 @@
+# See the OWNERS docs: https://go.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+  - MadhavJivrajani
+  - rajaskakodkar
```

Comment:
@rajaskakodkar will need to join k-sigs to be included in the OWNERS file.

Reply:
Ah, good catch
Thanks!